src/main/java/mate/academy/dto/item/CreateCartItemRequestDto.java

```diff
@@ -0,0 +1,14 @@
+package mate.academy.dto.item;
+
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
+
+public record CreateCartItemRequestDto(
+        @NotNull
+        @Min(0)
+        Long bookId,
+        @NotNull
+        @Min(0)
+        int quantity
```

Comment:
int type can't be null

```suggestion
        @Positive
        Long bookId,
        @Positive
        int quantity
```